spk/minio/src/minio.sc

```diff
@@ -0,0 +1,5 @@
+[minio]
+title="Minio"
```

Comment:
change to MinIO

Reply:
Fixed in https://github.com/SynoCommunity/spksrc/pull/3689/commits/f9277b298ab395637df41944d3a2b8422d28e2c3